src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public enum Colors {
+
+    RED,
+    GREEN,
+    BLUE,
+    YELLOW,
+    value;
```

Comment:
The enum constant `value` doesn't follow the naming convention for enums where the names are usually all uppercase. Please consider renaming it to follow the convention. For example, `VALUE` if it's meant to be a constant, or give it a meaningful name that represents an actual color if `value` is a placeholder.